pipestat/pipestat.py

```diff
@@ -624,3 +764,31 @@ def status_schema_source(self) -> Dict:
             the pipeline status structure
         """
         return self.get(STATUS_SCHEMA_SOURCE_KEY)
+
+
+def get_file_for_table(prj, pipeline_name, appendix=None, directory=None):
```

Comment:
this seems to belong in a `report.py` file, maybe?

Reply:
Done!